_schemaV2.graphql

```diff
@@ -11810,6 +11810,19 @@ type Partner implements Node {
   displayFullPartnerPage: Boolean
   displayWorksSection: Boolean
   distinguishRepresentedArtists: Boolean
+
+  # Return partner documents if current user has CMS access.
+  documentsConnection(
+    after: String
+    artistID: String
+    before: String
+
+    # Filter documents by ID.
+    documentIDs: [String]
+    first: Int
+    last: Int
+    showID: String
+  ): PartnerDocumentsConnectionConnection
```

Comment:
double `Connection` in the type name, can we get rid of this?

Reply:
Updated with https://github.com/artsy/metaphysics/pull/4489/commits/c4ecc372fa614158a3cf9b1f9705c2b249c67b11!